content/en/docs/instrumentation/python/automatic/agent-config.md

```diff
@@ -28,49 +28,30 @@ opentelemetry-instrument \
 
 Here's an explanation of what each configuration does:
 
-- `traces_exporter` specifies which trace exporter to use. In this case, traces
-  are being exported to `console` (stdout) and to `otlp`. The `otlp` option
-  tells `opentelemetry-instrument` to send it to an endpoint that accepts OTLP
-  via gRPC. The full list of available options for traces_exporter can be found
+- `traces_exporter` specifies which traces exporter to use. In this case, traces
+  are being exported to `console` (stdout) and with `otlp`. The `otlp` option
+  tells `opentelemetry-instrument` to send the traces to an endpoint that accepts OTLP
+  via gRPC. In order to use HTTP instead of gRPC, add `--exporter_otlp_protocol http`.
+  The full list of available options for traces_exporter can be found
   [here](https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/opentelemetry-instrumentation).
-- `otlp` used above for `traces_exporter` is the equivalent of using
-  `otlp_proto_grpc`. To send traces via HTTP instead of gRPC, replace
-  `otlp_proto_grpc` (or `otlp`) with `otlp_proto_http`.
 - `metrics_exporter` specifies which metrics exporter to use. In this case,
   metrics are being exported to `console` (stdout). It is currently required for
   your to specify a metrics exporter. If you aren't exporting metrics, specify
   `none` as the value instead.
```

Comment:
Not sure if it's still required to set a metrics exporter (probably not). Could also remove this part.

Reply:
At least as of last December it was needed (in my own testing). Otherwise you'd get an error at runtime. I'd love it if this isn't required anymore, though!